extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -176,6 +176,14 @@ export class ResourceTypePickerDialog extends DialogBase {
 
 	private selectResourceType(resourceType: ResourceType): void {
 		this._selectedResourceType = resourceType;
+		//handle special case when resource type is SQL Azure Database
+		if (this._selectedResourceType.name === 'sql-azure-setup') {
```

Comment:
I'd rather see this be an optional config field in the JSON (defaulting to Select if nothing is provided)